engine/src/witness/sol/program_swaps_witnessing.rs

```diff
@@ -217,10 +217,6 @@ pub async fn get_program_swaps(
 		.await;
 
 	Ok((new_swaps, closed_accounts))
-
-	// TODO: When submitting data we could technically submit the slot when the SwapEvent was
-	// queried for the new opened accounts. However, it's just easier to submit the slot when the
-	// SwapEndpointDataAccount was queried for both closed accounts and new opened accounts.
```

Comment:
Removing this comment as we've now added the `slot` to the Swap Event account data (on-chain) so there is no longer a question of what slot we should be submitting.